tests/serverpod_test_server/test_integration/test_tools/database_operations_in_endpoint_test.dart

```diff
@@ -82,4 +82,327 @@ void main() {
     },
     runMode: ServerpodRunMode.production,
   );
+
+  withServerpod(
+    'Given TestToolsEndpoint and rollbackDatabase afterEach',
+    (endpoints, session) {
+      group('when calling createSimpleDatasInsideTransactions', () {
+        setUpAll(() async {
+          await endpoints.testTools
+              .createSimpleDatasInsideTransactions(session, 123);
+        });
+
+        test("then finds SimpleDatas using the test session's transaction",
```

Comment:
Suggestion: Remove "using the test session's transaction".

Since we are not specifically fetching the transaction in the test, this is more of an implementation detail. We just want to make sure that we can read the result of what has happened inside a transaction in an endpoint right?